src/main/java/datawave/microservice/authorization/federation/config/FederatedAuthorizationServiceProperties.java

```diff
@@ -0,0 +1,117 @@
+package datawave.microservice.authorization.federation.config;
+
+import java.util.concurrent.TimeUnit;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.PositiveOrZero;
+
+public class FederatedAuthorizationServiceProperties {
+    private String federatedAuthorizationUri = "https://authorization:8443/authorization/v2";
+    
+    // max bytes to buffer for each rest call (-1 is unlimited)
+    private int maxBytesToBuffer = -1;
+    
+    @Valid
+    private RetryTimeoutProperties listEffectiveAuthorizations = new RetryTimeoutProperties();
```

Comment:
Maybe the field name would be clearer if it had properties in its name.

Reply:
whatever name you use here is what you will have to write out in the yaml configuration.  my feeling is that adding 'properties' would make this a bit verbose - especially if we did that throughout.

Reply:
whoah...  looking at this again and now i see the obvious mistake you were trying to point out.  yikes!  i think i fell victim to the overzealous-ness of intellij's rename feature.

Reply:
changed the naming.  hopefully that's better.